### ISSUE TITLE
z-index controls confusing

### ISSUE THREAD

**Issue by waynegraham** [scholarslab/Neatline#298]
The z-index control can create some confusion to end users. We need to look at a way to make it clearer what's going on; perhaps some method of visually shifting the layer order since the z-index is really an arbitrary number.

**Comment by davidmcclure** [comment]
Yeah, zindex is pretty technical - and, to make things worse, it doesn't even really work the way it does in CSS. Under the hood, OpenLayers implements zindex as a series of "steps" up or down, where layers get raised or lowered some number of times. The problem, though, is that it will stop performing these steps once a layer has hit the top or bottom of the stack, which means, bizarrely, that zindex values that are larger than the total number of layers on the map aren't guaranteed to be rendered.

For example, imagine the map has 5 layers. If Layer1 has a zindex of 3, and Layer2 has a zindex of 2, then Layer1 will be correctly rendered on top of Layer2. But, if Layer1 has a zindex of 101, and Layer2 has a zindex of 100, both layers will hit the "top" of the stack when OpenLayers tries to manifest the index, and Layer 2 will actually be rendered on top of Layer1 (assuming it was created after Layer1 chronologically, and appears second in the JSON that comes back from the server), since OpenLayers will simply push both as high as possible. Since Layer2 shows up second, it will win out, even though its zindex is smaller in an absolute sense.

Anyway - I've banged my head against the keyboard trying to fix this, and I don't believe there's a way to reliably enforce "real" CSS-style zindexes in OpenLayers 2.x. Hopefully this is fixed in OL3, and I wonder if this is an issue that we'll need to punt on until we move to that.
